2-alcance/README.md

```diff
@@ -1,3 +1,187 @@
-# Plano de alcance
+# PLANO DE ALCANCE
 
-## Referencias
\ No newline at end of file
+## INTRODUCCIÓN
+
+En el siguiente apartado, se va a especificar la estructura del alcance que tendrá el sitio web. El contenido se dividirá en dos fragmentos principales, como son las especificaciones funcionales y los requerimientos de contenido.
+
+## ESPECIFICACIONES FUNCIONALES
+Las especificaciones funcionales hacen referencia al comportamiento de, en este caso, una página web relacionada con la igualdad de género. La página web se llama Hoyportimañanapormi.com y en ella se encuentra contenido de todo tipo: noticias, reportajes, entrevistas, artículos, historias, consejos, etc.
```

Comment:
Si tenéis una sección relativa al contenido, dejad las referencias al mismo ("contenido de todo tipo") para entonces.